### ISSUE TITLE
Feature request: Add Linear and Logistic regression

### ISSUE THREAD

**Issue by dermatologist** [dermatologist/crisp-t#18]
## Feature request: Add Linear and Logistic regression 

CRISP-T supports machine learning methods except multiple linear regression and logistic regression using the library sklearn. Add a function in ml.py named get_regression(self, y: str) that takes the outcome variable as the single input. Use corpus and _process_xy to get data. Fit a logistic regression model if the outcome is binary. Otherwise, fit a linear regression model. Print relevant metrics such as MSE, accuracy, coefficients and intercept. Also, add these parameters to the corpus metadata. Follow the same pattern as in other functions currently in this file.

Next, add relevant tests to test_ml.py and make sure that the tests pass.

Then add a command-line option in cli.py to run this. Also, add it to the –ml option that runs all ML functions.

Finally, update the README.md and notes/INSTRUCTIONS.md with the newly added function.
